From: Christian Kastner <ckk@kvr.at>
Date: Mon, 19 May 2025 13:05:49 +0200
Subject: Drop non-DFSG-free server frontend

We cannot embed the default, React-based server frontend because we
cannot rebuild it from source, as not all relevant dependencies are
available in Debian.

Forwarded: not-needed
---
 tools/ui/CMakeLists.txt | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/tools/ui/CMakeLists.txt b/tools/ui/CMakeLists.txt
index 9687ca9..6dc8d26 100644
--- a/tools/ui/CMakeLists.txt
+++ b/tools/ui/CMakeLists.txt
@@ -21,9 +21,6 @@ if(LLAMA_BUILD_WEBUI OR LLAMA_BUILD_UI)
     endif()
 
     set(PUBLIC_ASSETS
-        index.html
-        bundle.js
-        bundle.css
         loading.html
     )
 
